bpf/lib/l3.h

```diff
@@ -100,6 +106,11 @@ static __always_inline int ipv6_local_delivery(struct __ctx_buff *ctx, int l3_of
 }
 #endif /* ENABLE_IPV6 */
 
+/* Performs IPv4 L3 handling and deliver the packet to the destination pod on
```

Comment:
Nit: I think you meant L2 handling?

Reply:
I was referring to `ipv4_l3` (decrease TTL and switch MAC addresses). I'm happy to refer to that under another name if we think it makes more sense.

Reply:
OK, then let's s/L3 handling/L2 and L3 handling/

Reply:
Fixed as well as a typo on `deliver[s]`.